.changeset/upset-tires-shout.md

```diff
@@ -0,0 +1,20 @@
+---
+'@astrojs/markdoc': minor
+'@astrojs/preact': minor
+'@astrojs/svelte': minor
+'@astrojs/react': minor
+'@astrojs/solid-js': minor
+'@astrojs/mdx': minor
+'@astrojs/vue': minor
+'create-astro': minor
+'@astrojs/prism': minor
+'@astrojs/telemetry': minor
+'@astrojs/upgrade': minor
+'astro': minor
+---
+
+Increases minimum Node version to 18.20.8
+
+Node 18 has now reached end-of-life and should not be used. For now, Astro will continue to support Node 18.20.8, which is the final LTS release of Node 18, as well as Node 20 and Node 22 or later. We will drop support for Node 18 in a future release, so we recommend upgrading to Node 22 as soon as possible. See Astro's [Node.js support policy](https://docs.astro.build/en/upgrade-astro/#support) for more details.
```

Comment:
Shouldn't we recommend v20 instead of v22?

Reply:
I mean since v18 reaches is EOL, shouldn't we only recommended to update to the next supported version, ie. 20? I don't mind fyi, just curious about the reason

Reply:
I suggested 22 because it's the current active LTS version, and avoids them needing to go through this all again too soon.